corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -67,6 +67,20 @@ FormplayerFrontend.reqres.setHandler('clearMenu', function () {
     $('#menu-region').html("");
 });
 
+$(document).bind("ajaxStart", function(){
+    var requestElements = $(".formplayer-request");
+    for (var i = 0; i < requestElements.length; i++ ) {
+        requestElements[i].style.pointerEvents = 'none';
+    }
```

Comment:
the more concise and cross browser way to do this is:

`$('.formplayer-request').css('pointer-events', 'none');`
http://api.jquery.com/css/